commands/orchestrationcommands/stackeventcommands/listresource.go

```diff
@@ -0,0 +1,154 @@
+package stackeventcommands
+
+import (
+	"strings"
+
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/commands/orchestrationcommands/stackcommands"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	"github.com/rackspace/rack/internal/github.com/fatih/structs"
+	osStackEvents "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stackevents"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stackevents"
+	"github.com/rackspace/rack/util"
+)
+
+var listResource = cli.Command{
+	Name:        "list-resource",
```

Comment:
Hmm, I know the docs have this categorized under "Stack Events", but it seems to me like it belongs with "Stack Resources":
current: `rack orchestration stack-event list-resource`
suggested: `rack orchestration stack-resource list-events`

What do you think?